web/templates/topic_sidebar_actions.hbs

```diff
@@ -13,14 +13,14 @@
             {{#unless topic_unmuted}}
                 <li class="hidden-for-spectators">
                     <a tabindex="0" class="sidebar-popover-unmute-topic" data-stream-id="{{ stream_id }}" data-topic-name="{{ topic_name }}">
-                        <i class="zulip-icon zulip-icon-mute" aria-hidden="true"></i>
+                        <i class="zulip-icon zulip-icon-unmute" aria-hidden="true"></i>
                         {{t "Unmute topic (muted stream)"}}
                     </a>
                 </li>
             {{else}}
                 <li class="hidden-for-spectators">
                     <a tabindex="0" class="sidebar-popover-remove-unmute" data-stream-id="{{ stream_id }}" data-topic-name="{{ topic_name }}">
-                        <i class="zulip-icon zulip-icon-mute" aria-hidden="true"></i>
+                        <i class="zulip-icon zulip-icon-unmute" aria-hidden="true"></i>
```

Comment:
Is it intended that we use the new icon in muted streams too?

Reply:
For topics in an unmuted stream, for the mute/unmute option mute icon is used. (same as before)
For topics in a muted stream, for the mute/unmute option unmute icon will be used. (which will be the only mute/unmute option in the muted stream once #25082 gets merged)

The above behavior is consistent everywhere i.e. in recent conversations, the message header, and the `mute/unmute` option in the topic sidebar.